rustls/src/msgs/handshake.rs

```diff
@@ -1,15 +1,16 @@
-use crate::key;
+use crate::enums::{CipherSuite, ProtocolVersion, SignatureScheme};
 use crate::msgs::base::{Payload, PayloadU16, PayloadU24, PayloadU8};
 use crate::msgs::codec;
 use crate::msgs::codec::{Codec, Reader};
 use crate::msgs::enums::ECCurveType;
+use crate::msgs::enums::HandshakeType;
```

Comment:
Style nit: I think it'd be better to concentrate `msgs::enums` imports in as few lines as possible.